library/core/src/main/java/com/google/android/exoplayer2/text/ssa/SsaStyle.java

```diff
@@ -207,6 +215,36 @@ private static float parseFontSize(String fontSize) {
     }
   }
 
+  private static boolean parseBold(String bold) {
+    try {
+      int boldFlag = Integer.parseInt(bold);
+      if(boldFlag == 1 || boldFlag == -1){
```

Comment:
Based on the specs, only -1 means true, or am I missing something? Also you could simplify it by inlining the return like `return Integer.parseInt(bold) == -1;`. Same for `parseItalic`.

Reply:
Yeah based on the specs it should only be -1 but when I was testing with vlc they took 1 as well. Do you think we should go for a clean implementation or a more forgiving one?

Reply:
I've also checked the libass implementation and it seems they are doing the same (-1, 1): https://github.com/libass/libass/blob/82b225b3d6653091d028b39d561d185ed76a7be5/libass/ass_parse.c#L118

So IMO we can go with the more forgiving implementation, but lets wait for @icbaker 's confirmation.

Reply:
+1 to more forgiving

though weirdly for italic it seems libass **only** consider 1 to mean `true` (just below the line @szaboa linked), while the spec only allows `-1`...? P

Maybe it would be easier just to do `!= 0`?